updated airgapped instructions to use `hauler` (#139)
neuvector/docs: versioned_docs/version-5.4/02.deploying/09.airgap/09.airgap.md

---
title: Airgapping NeuVector
taxonomy:
    category: docs
slug: /deploying/airgap
---

> ### Prerequistes

Reviewer: ```suggestion
### Prerequisites
```

We need to install two tools to easily airgap everything for NeuVector...

* [Hauler](https://github.com/hauler-dev/hauler) - Airgap Swiss Army Knife
* [Helm](https://github.com/helm/helm) - The Kubernetes Package Manager

```bash
# install hauler
curl -sfL https://get.hauler.dev | bash

# install helm
curl -fsSL https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
```

### Fetch the Images and Helm Chart

> In order to get the required container images and helm chart for NeuVector, we are able to utilize `hauler` to easily fetch, package, and distribute all the bits. We can utilize commands or declarative manifests within `hauler` to do this... feel free to use either method...

Reviewer: ```suggestion
In order to get the required container images and Helm chart for NeuVector, we are able to utilize `hauler` to easily fetch, package, and distribute all the bits. We can utilize commands or declarative manifests within `hauler` to do this using either method.
```

#### For Supported Customers...

> If you are a **Rancher Prime** or **Rancher Government Carbide** customer, please make sure to use the commands that fetch the all the artifacts from the respective private registry. If you're a supported customer and do not know the correct `<registry-url>` for Prime or Carbide, please contact your respective support team!

Reviewer: ```suggestion
If you are a **Rancher Prime** or **Rancher Government Carbide** customer, please make sure to use the commands that fetch all artifacts from the respective private registry. If you're a supported customer and do not know the correct `<registry-url>` for Prime or Carbide, please contact your respective support team!
```
